### ISSUE TITLE
Cutter Crash on latest build 

### ISSUE THREAD

**Issue by securitykitten** [radareorg/cutter#1204]
**Environment information**
* Operating System: Fedora 29 x64
* Cutter version: 1.7.4  (git pull `a8fbe72bf76739b342f0570e4844f32de050112a`) Built from latest git
* File format: NES ROM (Doesn't seem to matter on this bug)

**Describe the bug**
When adding an extra graph window, Cutter will crash.

**To Reproduce**
1. Load binary file [hackingtime.zip](https://github.com/radareorg/cutter/files/2880619/hackingtime.zip)
2. Click on "Windows" -> "Add Extra" -> Graph View
3. Cutter will crash. With the following log.
```
$ /usr/bin/Cutter 
Warning: Ignoring XDG_SESSION_TYPE=wayland on Gnome. Use QT_QPA_PLATFORM=wayland to run on Wayland anyway.
Python init
[x] Analyze all flags starting with sym. and entry0 (aa)
[x] Finding xrefs in noncode section with anal.in = 'io.maps'
[x] Analyze value pointers (aav)
[x] Value from 0x00008000 to 0x00010000 (aav)
[x] 0x00008000-0x00010000 in 0x8000-0x10000 (aav)
[[I 10:05:15.861 CutterNotebookApp] 302 GET /?token=e0efa5584557748d25909df649c2db602890694ec4e55c05 (::1) 0.91ms
[x] Emulate code to find computed references (aae)
[x] Analyze function calls (aac)
[x] Analyze len bytes of instructions for references (aar)
[x] Speculatively constructing a function name for fcn.* and sym.func.* functions (aan)
[x] Type matching analysis for all functions (aaft)
[x] Use -AA or aaaa to perform additional experimental analysis.

Task 1 finished

Task 2 finished
No virtual tables found
Received signal 11 SEGV_MAPERR 000000000008
#0 0x7fdda191ea4f <unknown>
#1 0x7fdda191ee4b <unknown>
#2 0x7fdda191f4ce <unknown>
#3 0x7fdd9d00e030 <unknown>
#4 0x7fdda6c02fa0 <unknown>
#5 0x0000004e03e5 DisassemblerGraphView::DisassemblerGraphView()
#6 0x0000005af42b GraphWidget::GraphWidget()
#7 0x0000004fde65 MainWindow::on_actionExtraGraph_triggered()
#8 0x0000005e8fb1 MainWindow::qt_metacall()
#9 0x7fdd9d34fa14 QMetaObject::activate()
#10 0x7fdda6be0b62 QAction::triggered()
#11 0x7fdda6be2f80 QAction::activate()
#12 0x7fdda6d4bcba QMenuPrivate::activateCausedStack()
#13 0x7fdda6d52dc6 QMenuPrivate::activateAction()
#14 0x7fdda6d53df9 QMenu::mouseReleaseEvent()
#15 0x7fdda6c24828 QWidget::event()
#16 0x7fdda6d5630b QMenu::event()
#17 0x7fdda6be6b3c QApplicationPrivate::notify_helper()
#18 0x7fdda6bee30f QApplication::notify()
#19 0x7fdd9d323e28 QCoreApplication::notifyInternal2()
#20 0x7fdda6bed24a QApplicationPrivate::sendMouseEvent()
#21 0x7fdda6c3e730 QWidgetWindow::handleMouseEvent()
#22 0x7fdda6c40eeb QWidgetWindow::event()
#23 0x7fdda6be6b3c QApplicationPrivate::notify_helper()
#24 0x7fdda6bedd30 QApplication::notify()
#25 0x7fdd9d323e28 QCoreApplication::notifyInternal2()
#26 0x7fdd9e888767 QGuiApplicationPrivate::processMouseEvent()
#27 0x7fdd9e889d15 QGuiApplicationPrivate::processWindowSystemEvent()
#28 0x7fdd9e8659fb QWindowSystemInterface::sendWindowSystemEvents()
#29 0x7fdd8c49681a xcbSourceDispatch()
#30 0x7fdd99f2c06d g_main_context_dispatch
#31 0x7fdd99f2c438 <unknown>
#32 0x7fdd99f2c4d0 g_main_context_iteration
#33 0x7fdd9d37b1ef QEventDispatcherGlib::processEvents()
#34 0x7fdd9d32278a QEventLoop::exec()
#35 0x7fdd9d32b190 QCoreApplication::exec()
#36 0x0000004b557d main
#37 0x7fdd9cb20413 __libc_start_main
#38 0x0000004bb4ee _start
  r8: 0000000000000000  r9: 000000000000001d r10: 0000000000ed6530 r11: 0000000000000000
 r12: 0000000006bcb750 r13: 00000000006bcce0 r14: 00000000055a5730 r15: 00000000055a5730
  di: 0000000000000000  si: 0000000000000000  bp: 00007fffd804d930  bx: 0000000006bcb5d0
  dx: 0000000000000000  ax: 0000000000000000  cx: 0000000000000000  sp: 00007fffd804d7c8
  ip: 00007fdda6c02fa0 efl: 0000000000010246 cgf: 002b000000000033 erf: 0000000000000004
 trp: 000000000000000e msk: 0000000000000000 cr2: 0000000000000008
[end of stack trace]
Calling _exit(1). Core file will not be generated.
```

**Comment by ITAYC0HEN** [comment]
Pull Request #1202 is already waiting to get merged and will fix this
crash. Thank you for reporting :)

**Comment by securitykitten** [comment]
Awesome, didn't see the PR!  Thanks

**Comment by ITAYC0HEN** [comment]
Closed at #1202
